### DISCUSSION TITLE
How we can get the pth files for this model crnn_vgg16_bn

### DISCUSSION THREAD

**Discussion by Shiva-Ai4m** [mindee/doctr#1187] [General]
I need .pth file for this model crnn_vgg16_bn which can be used in another codes ... Can anyone help me out with this  Please??

**Comment by felixT2K** [comment]
`wget -O crnn_vgg16_bn.pt "https://doctr-static.mindee.com/models?id=v0.3.1/crnn_vgg16_bn-9762b0b0.pt&src=0"`
